region/norfair/east.json

```diff
@@ -211,6 +211,27 @@
                     {"heatFrames": 350}
                   ]
                 },
+                {
+                  "name": "Sova Boost",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canUseEnemies",
+                    "Morph",
```

Comment:
I think canDamageBoost should be here instead of canUseEnemies. I know it's kind of trivial, but I think that's what's been done before (at least for the West Ocean Bug Boost).